Cargo.toml

```diff
@@ -302,6 +302,7 @@ glob = "0.3.1"
 half = "2.4.1"
 hostname = "0.4"
 iana-time-zone = "0.1.57"
+icu = "2.0.0"
```

Comment:
Folks are probably going to complain less about the number of crates and compile times if you use `icu_collator` instead of the entire meta crate.